inngest/src/main/kotlin/com/inngest/InngestFunctionConfigBuilder.kt

```diff
@@ -9,7 +9,7 @@ import java.time.Duration
 // TODO: Throw illegal argument exception
 class InngestFunctionConfigBuilder {
     var id: String? = null
-    private var name: String? = null
+    internal var name: String? = null
```

Comment:
this is because you use it to build the function name for the failure function, right?

Reply:
exactly yeah